conda/gateways/connection/adapters/jlap/repodata_proxy.py

```diff
@@ -257,23 +189,30 @@ class FileResponse(NamedTuple):
 
 
 def static_file_headers(
-    filename, root, mimetype="auto", download=False, charset="UTF-8"
+    filename: str,
+    root: str,
+    mimetype: str = "auto",
+    download: bool = False,
+    charset: str = "UTF-8",
```

Comment:
"utf-8"

Reply:
this code is copied straight from bottle, and we should remove it.

Reply:
That makes sense. It will most likely disappear the further we refine this code.